src/config/dynamic-theme-fixes.config

```diff
@@ -10200,11 +10200,23 @@ img[src*="hashpower@2x]
 nirsoft.net
 
 CSS
-.menub1 a:link, .menub1 a:visited{
-    color: #0040ff;
+a:link {
+    color: ${#0AA};
```

Comment:
I'm not amused by this one, this changes the color to green, which doesn't represent the original site.